osquery/utils/conversions/windows/strings.cpp

```diff
@@ -31,6 +35,50 @@ std::wstring stringToWstring(const std::string& src) {
   return utf16le_str;
 }
 
+LONGLONG cimDatetimeToUnixtime(const std::string& src) {
+  // First init the SWbemDateTime class
+  ISWbemDateTime* pCimDateTime = nullptr;
+  auto hres = CoCreateInstance(CLSID_SWbemDateTime,
```

Comment:
What are the allocation/deallocation expectations for this?

Reply:
[According to the MSDN](https://docs.microsoft.com/en-us/windows/win32/api/combaseapi/nf-combaseapi-cocreateinstance) we don't need to worry about allocation/deallocation when using this explicit API call:

>The CoCreateInstance function provides a convenient shortcut by connecting to the class object associated with the specified CLSID, creating an uninitialized instance, and releasing the class object. As such, it encapsulates the following functionality:
```
CoGetClassObject(rclsid, dwClsContext, NULL, IID_IClassFactory, &pCF); 
hresult = pCF->CreateInstance(pUnkOuter, riid, ppvObj) 
pCF->Release();
```